chia/timelord/timelord.py

```diff
@@ -100,10 +125,20 @@ async def _start(self):
             self.config["vdf_server"]["port"],
         )
         self.last_state: LastState = LastState(self.constants)
-        if not self.sanitizer_mode:
+        slow_bluebox = self.config.get("slow_bluebox", False)
+        if not self.bluebox_mode and not slow_bluebox:
```

Comment:
if self.bluebox_mode is False, but slow_bluebox is True, we should **not** enter bluebox mode. This is pretty confusing since someone might accidentally go into bluebox without intending to.

Reply:
we enter `_manage_chains` - main algorithm only if _both_ `bluebox_mode` and `slow_bluebox` are False. every other case goes to the else loop that guarantees to bluebox.

Reply:
fixed, I see the point now - to always force bluebox_mode as True in order to bluebox.